src/test/java/seedu/nutrigoals/logic/LogicManagerTest.java

```diff
@@ -65,8 +65,8 @@ public void execute_commandExecutionError_throwsCommandException() {
 
     @Test
     public void execute_validCommand_success() throws Exception {
-        String listCommand = ListCommand.COMMAND_WORD;
-        assertCommandSuccess(listCommand, ListCommand.MESSAGE_SUCCESS, model);
```

Comment:
Is it better to keep this since `list` can be used without any arguments?

Reply:
`list` can be used without arguments, but in that case `MESSAGE_SUCCESS` will have to contain the current day as well. I replaced it with `ClearCommand` because it does not require any extra information.

I have tested `list` in the `ListCommandTest` class! :smiley: